ExtensionFrSexEst.owl

```diff
@@ -648,6 +829,42 @@
                 <owl:someValuesFrom rdf:resource="http://www.semanticweb.org/martin/ontologies/2016/8/untitled-ontology-26#Investigation.FrSexEst"/>
             </owl:Restriction>
         </rdfs:subClassOf>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://w3id.org/rdfbones/core#isMentioned"/>
+                <owl:hasValue rdf:resource="http://www.semanticweb.org/martin/ontologies/2016/8/untitled-ontology-26#&apos;Acsádi_&amp;_Nemeskéri_1970&apos;"/>
+            </owl:Restriction>
+        </rdfs:subClassOf>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://purl.obolibrary.org/obo/OBI_0000293"/>
+                <owl:qualifiedCardinality rdf:datatype="http://www.w3.org/2001/XMLSchema#nonNegativeInteger">1</owl:qualifiedCardinality>
+                <owl:onClass rdf:resource="http://www.semanticweb.org/martin/ontologies/2016/8/untitled-ontology-26#DegreeOfSexualization"/>
+            </owl:Restriction>
+        </rdfs:subClassOf>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://purl.obolibrary.org/obo/OBI_0000299"/>
+                <owl:qualifiedCardinality rdf:datatype="http://www.w3.org/2001/XMLSchema#nonNegativeInteger">1</owl:qualifiedCardinality>
+                <owl:onClass rdf:resource="http://www.semanticweb.org/martin/ontologies/2016/8/untitled-ontology-26#EstimatedSex"/>
+            </owl:Restriction>
+        </rdfs:subClassOf>
+        <core1:description>Positive values of the DegreeOfSexualization indicate masculinity while negative values of the DegreeOfSexualization indicate femininity.
+If the DegreeOfSexualization ranges between -0.4 and 0.4 sex estimation is regareded as uncertain.</core1:description>
+    </owl:Class>
+    
+
+
+    <!-- http://www.semanticweb.org/martin/ontologies/2016/8/untitled-ontology-26#EstimatedSex -->
+
+    <owl:Class rdf:about="http://www.semanticweb.org/martin/ontologies/2016/8/untitled-ontology-26#EstimatedSex">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/OBI_0000938"/>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://www.semanticweb.org/martin/ontologies/2016/8/untitled-ontology-26#&apos;inverse_IsAbout&apos;"/>
+                <owl:hasValue rdf:resource="http://www.semanticweb.org/martin/ontologies/2016/8/untitled-ontology-26#SpecificationOfDataToBeGeneratedInAnInvestigation.FrSexEst"/>
+            </owl:Restriction>
+        </rdfs:subClassOf>
     </owl:Class>
```

Comment:
The labels for this categorical measurement datum are currently defined as instances of this class. What should be the instances are the individual measurements undertaken during the investigation. The labels are to be defined as instances of some subclass of obo:OBI_0000963 ('Categorical label') and referred to with the property obo:OBI_0000999 ('has categorical label'). See also RDFBones-O wiki page [Data in OBI](https://github.com/RDFBones/RDFBones-O/wiki/Data-in-OBI).